README.md

```diff
@@ -84,6 +84,10 @@ For more API calls and environment variables, see [USAGE.md](docs/USAGE.md).
 
 To see what you can do with **OpenVINO™ integration with TensorFlow**, explore the demos located in the [examples](./examples) directory.  
 
+## Docker Support
+Dockerfiles for Ubuntu* 18.04, Ubuntu* 20.04, and TensorFlow* Serving are provided which can be used to build runtime Docker* images for **OpenVINO™ integration with TensorFlow** on CPU, GPU, VPU, and VAD-M. 
+For more details see [docker readme](docker/README.md).
+
 ## Try it on Intel<sup>®</sup> DevCloud
```

Comment:
Can we also add links to the prebuilt Docker images in the main ReadMe? I think there will be more developer interest in using the prebuilt images directly instead of building them from source.

Reply:
Links added to main README